src/machine/machine_rp2040_adc.go

```diff
@@ -20,43 +39,112 @@ func InitADC() {
 	waitForReady()
 }
 
-// Configure configures a ADC pin to be able to be used to read data.
+// Below methods are for standard tinygo, ADC{} pin based access to the ADC peripheral.
```

Comment:
The comment on this function should follow standard [Go documentation convention](https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#doc-comments)

Reply:
Will tidy up.